modules/tracking/tutorials/tutorial_customizing_cn_tracker.markdown

```diff
@@ -0,0 +1,73 @@
+Customizing the CN Tracker {#tutorial_customizing_cn_tracker}
+======================
+
+Goal
+----
+
+In this tutorial you will learn how to
+
+-   Set custom parameters for CN tracker.
+-   Use your own feature-extractor function for the CN tracker.
+
+This document contains tutorial for @ref cv::TrackerKCF.
+
+Source Code
+-----------
+
+@includelineno tracking/samples/tutorial_customizing_cn_tracker.cpp
+
+Explanation
+-----------
+
+This part explains how to set custom parameters and use your own feature-extractor function for the CN tracker.
+If you need more detailed tutorial about the basic things, please refer to @ref tutorial_introduction_to_tracker.
```

Comment:
instead of about the basic things, I would say about an introduction to tracking or so.